planning/README.md

```diff
@@ -0,0 +1,53 @@
+# Planning
+
+## Group Rules HYF-Group2
+
```

Comment:
Hi :blush: ,Can you add group rules? for example: finishing taks on time, informing team member about what s/he did, help each other, being respectful etc.

Reply:
okey noproblem i will add them

Reply:
ok, thank you :hugs: